dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/DatadogGesturesTracker.kt

```diff
@@ -46,6 +46,29 @@ internal class DatadogGesturesTracker(
 
     // endregion
 
+    // region Object
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
I would really advise to properly test all these re-implementations of `equals/hashcode` methods ?